.github/workflows/test_destinations_local.yml

```diff
@@ -88,45 +172,40 @@ jobs:
           installer-parallel: true
           version: 1.8.5
 
-      - name: Load cached venv
-        id: cached-poetry-dependencies
-        uses: actions/cache@v3
-        with:
-          path: .venv
-          key: venv-${{ runner.os }}-${{ steps.setup-python.outputs.python-version }}-${{ hashFiles('**/poetry.lock') }}-local-destinations
-
       - name: Install dependencies
-        run: poetry install --no-interaction -E postgres -E postgis -E duckdb -E parquet -E filesystem -E cli -E weaviate -E qdrant -E sftp -E deltalake -E pyiceberg --with sentry-sdk,adbc,pipeline,ibis,adbc
-
-      - name: Upgrade sqlalchemy
-        run: poetry run pip install sqlalchemy==2.0.18  # minimum version required by `pyiceberg`
-
-      - name: Start SFTP server
-        run: docker compose -f "tests/load/filesystem_sftp/docker-compose.yml" up -d
+        run: poetry install --no-interaction --with sentry-sdk,pipeline,ibis,providers${{ matrix.with }} --extras "${{ matrix.extras }}"
 
-      - name: Configure SSH Agent for sftp tests
+      - name: Copy secrets for local tests
         run: |
-          mkdir -p /home/runner/.ssh
-          cp tests/load/filesystem_sftp/bootstrap/bobby_rsa /home/runner/.ssh/id_rsa
-          cp tests/load/filesystem_sftp/bootstrap/bobby_rsa.pub /home/runner/.ssh/id_rsa.pub
+          cp tests/.dlt/dev.secrets.toml tests/.dlt/secrets.toml
+
+      - name: Run post install commands
+        run: ${{ matrix.post_install_commands }}
+        if: ${{ matrix.post_install_commands }}
 
       # always run full suite, also on branches
       - name: Run tests Linux
         run: |
           eval "$(ssh-agent -s)"
-          poetry run pytest tests/load --ignore tests/load/sources
-          poetry run pytest tests/cli
-        env:
-          DESTINATION__POSTGRES__CREDENTIALS: postgresql://loader:loader@localhost:5432/dlt_data
```

Comment:
where did those go?

Reply:
to this new file, see comment above. there is one step in the workflow that copies these back in.